wallet/src/main/kotlin/net/thechance/wallet/api/controller/AdminWalletController.kt

```diff
@@ -0,0 +1,30 @@
+package net.thechance.wallet.api.controller
+
+import net.thechance.wallet.api.dto.balance.AdminDepositRequest
+import net.thechance.wallet.service.AdminWalletService
+import org.springframework.http.ResponseEntity
+import org.springframework.security.core.annotation.AuthenticationPrincipal
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+@RestController
+@RequestMapping("/wallet/admin")
+class AdminWalletController(
+    private val adminWalletService: AdminWalletService
+) {
+    @PostMapping("/deposit")
+    fun depositToUser(
```

Comment:
What about renaming each `deposit` word in the files to `depositMoney` for more clarity 

```suggestion
    fun depositMoneyToUser(
```